LupusNet/ETL/Gladel1.0/runETL.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -e
+
+#############
+## RUN ETL ##
+#############
+REGISTRY_ETL_RUNNER=harbor.honeur.org
+REPOSITORY_ETL_RUNNER=library
+IMAGE_ETL_RUNNER=etl-runner
+VERSION_ETL_RUNNER=1.1.4
+TAG_ETL_RUNNER=$VERSION_ETL_RUNNER
+
+LOG_FOLDER_HOST=${PWD}/log
+DATA_FOLDER_HOST=${PWD}/data
+QA_FOLDER_HOST=${PWD}/qa
+
+echo "Pull ETL runner image"
+docker pull $REGISTRY_ETL_RUNNER/$REPOSITORY_ETL_RUNNER/$IMAGE_ETL_RUNNER:$TAG_ETL_RUNNER
+
+REGISTRY=harbor.lupusnet.org
+REPOSITORY=etl-gladel-1.0
+CDM_SCHEMA=gladel1
+VOCAB_SCHEMA=omopcdm
+
+#echo "Download ETL questions"
+#curl -fsSL https://raw.githubusercontent.com/Feder8-Platform/Feder8-Setup/main/LupusNet/ETL/Gladel1.0/questions.json --output questions.json
+
+touch etl-runner.env
+# etl image
+echo "THERAPEUTIC_AREA=lupus" >> etl-runner.env
+echo "REGISTRY=$REGISTRY" >> etl-runner.env
+echo "ETL_IMAGE_NAME=$REPOSITORY/etl" >> etl-runner.env
+echo "ETL_IMAGE_TAG=current" >> etl-runner.env
+# logs
+echo "LOG_FOLDER_HOST=$LOG_FOLDER_HOST" >> etl-runner.env
+echo "LOG_FOLDER=/log" >> etl-runner.env
+# schemas 
+echo "DB_SCHEMA=$CDM_SCHEMA" >> etl-runner.env
+echo "VOCAB_SCHEMA=$VOCAB_SCHEMA" >> etl-runner.env 
+# source data
+echo "DATA_FOLDER_HOST=$DATA_FOLDER_HOST" >> etl-runner.env
+echo "DATA_FOLDER=/source" >> etl-runner.env
+echo "SOURCE_RELEASE_DATE=$SOURCE_RELEASE_DATE" >> etl-runner.env
```

Comment:
This line can be removed as $SOURCE_RELEASE_DATE is not defined 
SOURCE_RELEASE_DATE will be set by the ETL runner by answering the interactive question